### ISSUE TITLE
Search services by language

### ISSUE THREAD

**Issue by yaoberh** [AwesomeTTS/awesometts-anki-addon#187]
Would be cool if a search bar or a language selector/filter could be implemented in the services window so one could choose which language one wants. This is especially useful for the ones of us who study multiple languages and want to make presets with all available voices for a language and then run a on-the-fly tts. Just thought this would be a cool addition.

**Comment by lucwastiaux** [comment]
Yes! This is super important now that the main services offer tons of voices. Note that language tools already does it this way (paid add-on https://ankiweb.net/shared/info/771677663)

**Comment by lucwastiaux** [comment]
I'm planning on reorganizing services in AwesomeTTS to make this easier.
